focus-android/gradle.properties

```diff
@@ -25,3 +25,5 @@ libUrl=https://github.com/mozilla-mobile/firefox-android/tree/main/focus-android
 libVcsUrl=https://github.com/mozilla-mobile/firefox-android.git
 libLicense=MPL-2.0
 libLicenseUrl=https://www.mozilla.org/en-US/MPL/2.0/
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
Are we enabling r8 full mode in focus?

Reply:
No, I tested and minifying with R8 full mode also crashes Focus. 
As for the sample apps they do not have minify enabled.

Reply:
Yes, that's what my understanding was. I see we've now added the property below 👍

Reply:
Yes, thanks for pointing that out!